tests/decomposition/test_mvica.py

```diff
@@ -30,15 +30,18 @@
 
 import pytest
 import numpy as np
-from mvlearn.decomposition.mv_ica import _hungarian
+from multiviewica import _hungarian
```

Comment:
what is multiviewica here ??? relative import with a missing . ?

Reply:
I think _hungarian is a function in Hugo's repo, but it should be private, so I think it could be avoided here. Or since its a simple function, it could also be copy-pasted.

Reply:
Ok for copy pasting

Reply:
yes please no other dependency

Reply:
Is this PR still going to add multiviewica as a dependency? It's in the requirements.txt and is imported in multiviewica.py

Reply:
Yes I would rather do that for now.